### ISSUE TITLE
Cell type report: Update CellAssign ridgeplot section

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/scpca-nf#518]
After further exploration of CellAssign probabilities when run on ScPCA data, we will circle back to updating this plot and associated text on how to interpret it.

Either way, we will want to bulletify the text!

**Comment by jashapiro** [comment]
Oooh... Somebody added `bounds` to `stat_density()` about a few years ago! https://github.com/tidyverse/ggplot2/pull/4013

Since I assume this is what is used by ridge plots under the hood, we should be able to take advantage of this. At least I hope we can.

**Comment by sjspielman** [comment]
From our discussion last week, we'll take the following steps here:

- Use faceted density plot with bounds & add a rug
  - With added rug, we probably/certainly don't need the line segments 
- Remove the FALSE distribution.

**Comment by sjspielman** [comment]
Well, not sure about density plots here... Here's with `geom_density(aes(y = ..scaled..))` (without `..scaled..` is much worse). The y-axis is cut off in my screen shot, but it's decimals between `[0,1]`.

<img width="450" alt="Screenshot 2023-10-31 at 10 52 29 AM" src="https://github.com/AlexsLemonade/scpca-nf/assets/4701111/a50a2828-2e32-4c09-956a-43d263705676">


<br><br><br>

Just playing around with different ideas, here's a violin plot scaled by width and I added a "rug" of blue points slightly below each violin: 
<img width="400" alt="Screenshot 2023-10-31 at 10 55 35 AM" src="https://github.com/AlexsLemonade/scpca-nf/assets/4701111/7a46ade7-ec74-4789-817f-fa3061cc3c0a">

**Comment by sjspielman** [comment]
Made lots of density plots with varying `adjust` values, [from docs](https://ggplot2.tidyverse.org/reference/geom_density.html) - 
> A multiplicate bandwidth adjustment. This makes it possible to adjust the bandwidth while still using the a bandwidth estimator. For example, adjust = 1/2 means use half of the default bandwidth.

I recommend downloading this file and zooming in to really see! I removed a lot of theme elements and made the rug red to see the distributions more clearly just for comparison purposes. I think `0.25` is maybe the best one, but that's for _this data._ I'm not actually sure how much we can generalize?

![adjusts](https://github.com/AlexsLemonade/scpca-nf/assets/4701111/88a50c3c-fd4c-40c0-b609-46fdd6ed3bee)

**Comment by jashapiro** [comment]
I don't think we want to use `adjust`, as we _don't_ want to use the automatic bandwidth estimator. We should use `bw` directly, which we can do knowing that we are only going on the range from 0 to 1. A reasonable value would probably be 0.1 or 0.05.

We probably also still want vertical stacking with `facet_grid(rows = label)`, or whatever the appropriate variable is.

**Comment by allyhawkins** [comment]
I would definitely stack so that each row is a cell type, but I also might change the color of the lines in the density and rug plot. With the black rug plot, it's hard to tell axes ticks vs. the rug when there are only a few cells.

**Comment by sjspielman** [comment]
> With the black rug plot, it's hard to tell axes ticks vs. the rug when there are only a few cells.

yep, that was just for choosing plot params! I ended up with `bw=0.03` and using blue for the rug, which was the most visible color I've tried. 

FWIW here's where I've gotten so far.. it's coming along, will file PR by end of day!

<img width="475" alt="Screenshot 2023-10-31 at 2 23 43 PM" src="https://github.com/AlexsLemonade/scpca-nf/assets/4701111/a3c1841c-1389-461c-ba54-88cd56c2e3fb">
